src/Core/tests/DeviceTests/Handlers/SearchBar/SearchBarHandlerTests.iOS.cs

```diff
@@ -135,6 +136,48 @@ public async Task CharacterSpacingInitializesCorrectly()
 			Assert.Equal(xplatCharacterSpacing, values.PlatformViewValue);
 		}
 
+		[Fact]
+		public async Task SearchBarFocusedIsWorking()
```

Comment:
Can we run the code through the handler and simplify this a bit to 

```C#
[Fact]
public async Task SearchBarFocusedIsWorking()
{
	var searchBar1 = new SearchBarStub
	{
		Text = "SearchBar 1"
	};

	await InvokeOnMainThreadAsync(async () =>
	{
		var contentViewHandler = CreateHandler(searchBar1);
		await contentViewHandler.PlatformView.AttachAndRun(async () =>
		{
			searchBar1.Handler.Invoke(nameof(IView.Focus), new FocusRequest(false));
			await searchBar1.WaitForFocused();
			Assert.True(searchBar1.IsFocused);

			searchBar1.Handler.Invoke(nameof(IView.Unfocus), new FocusRequest(false));
			await searchBar1.WaitForUnFocused();
			Assert.False(searchBar1.IsFocused);
		});
	});
}
```

Reply:
Added.